### PR TITLE
Currency : Fixing jenkins pipeline job  for nbconvert

### PR DESCRIPTION
**Pull request by stutiibm**
## Checklist

- [-x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [-x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: n/nbconvert/build_info.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "maintainer": "Vishaka.Desai@ibm.com",
+    "maintainer": "Stuti.Wali@ibm.com",
     "package_name": "nbconvert",
     "github_url": "https://github.com/jupyter/nbconvert",
     "version": "v7.7.3",
```
Comment (nikhil-kalbande) [inline n/nbconvert/build_info.json:5]: @stutiibm  change this to latest version you verified i.e. v7.12.0/

File: n/nbconvert/build_info.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "package_dir": "n/nbconvert/",
     "docker_build": false,
     
-    "v7.7.3": {
+    "v7.12.0": {
```
Comment (nikhil-kalbande) [inline n/nbconvert/build_info.json:11]: you can append new version string /pattern or add new entry. need to keep old version supported

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: n/nbconvert/build_info.json (hunk 2 above)
Comment (stutiibm) [reply n/nbconvert/build_info.json:11]: Okay

Comment (stutiibm) [reply n/nbconvert/build_info.json]: Made required changes.

**Pull request merged**